apache-hbase-mixin/alerts/alerts.libsonnet

```diff
@@ -0,0 +1,113 @@
+            ||| % $._config,
+            'for': '5m',
+            labels: {
+              severity: 'warning',
+            },
+            annotations: {
+              summary: 'There is a limited amount of heap memory available to the JVM.',
+              description:
+                (
+                  'The heap memory usage for the JVM on instance {{$labels.instance}} in cluster {{$labels.hbase_cluster}} is {{printf "%%.0f" $labels.value}} percent, which is above the threshold of %(alertsHighHeapMemUsage)s'
+                ) % $._config,
+            },
+          },
+          {
+            alert: 'HighNonHeapMemUsage',
+            expr: |||
+              100 * sum without(context, hostname, processname) (jvm_metrics_mem_non_heap_used_m / clamp_min(jvm_metrics_mem_non_heap_committed_m, 1))  > %(alertsHighNonHeapMemUsage)s
+            ||| % $._config,
+            'for': '5m',
+            labels: {
+              severity: 'warning',
+            },
+            annotations: {
+              summary: 'There is a limited amount of non-heap memory available to the JVM.',
+              description:
+                (
+                  'The non-heap memory usage for the JVM on instance {{$labels.instance}} in cluster {{$labels.hbase_cluster}} is {{printf "%%.0f" $labels.value}} percent, which is above the threshold of %(alertsHighNonHeapMemUsage)s'
+                ) % $._config,
+            },
+          },
+          {
+            alert: 'DeadRegionServer',
+            expr: |||
+              server_num_dead_region_servers > %(alertsDeadRegionServer)s
+            ||| % $._config,
+            'for': '5m',
+            labels: {
+              severity: 'warning',
+            },
+            annotations: {
+              summary: 'One or more RegionServer(s) has become unresponsive.',
+              description:
+                (
+                  '{{$labels.value}} RegionServer(s) in cluster {{$labels.hbase_cluster}} are unresponsive, which is above the threshold of %(alertsDeadRegionServer)s. The name(s) of the dead RegionServer(s) are {{$labels.deadregionservers}}'
+                ) % $._config,
+            },
+          },
+          {
+            alert: 'OldRegionsInTransition',
+            expr: |||
+              100 * assignment_manager_rit_count_over_threshold / clamp_min(assignment_manager_rit_count, 1) > %(alertsOldRegionsInTransition)s
+            ||| % $._config,
+            'for': '5m',
+            labels: {
+              severity: 'warning',
+            },
+            annotations: {
+              summary: 'RegionServers are in transition for longer than expected.',
+              description:
+                (
+                  '{{printf "%%.0f" $labels.value}} percent of RegionServers in transition in cluster {{$labels.hbase_cluster}} are transitioning for longer than expected, which is above the threshold of %(alertsOldRegionsInTransition)s'
```

Comment:
I'd either add the word "percent" or `%%s` to the end of this description since it is a percent threshold...also applies to the other alerts with percentage thresholds